cfme/tests/infrastructure/test_ssa_vddk.py

```diff
@@ -0,0 +1,84 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.common.vm import VM
+from cfme.configure.configuration.analysis_profile import AnalysisProfile
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.utils import conf
+from cfme.utils.hosts import setup_host_creds
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+from markers.env_markers.provider import ONE_PER_VERSION
+
+pytestmark = [
+    pytest.mark.tier(3),
+    test_requirements.smartstate,
+    pytest.mark.meta(server_roles="+smartproxy +smartstate"),
+    pytest.mark.provider([VMwareProvider],
+                         selector=ONE_PER_VERSION, scope='module'),
```

Comment:
REQUIRED: please follow our [style guide](http://cfme-tests.readthedocs.io/guides/dev_guide.html) and use one indent.